kafka_utils/kafka_cluster_manager/cluster_info/error.py

```diff
@@ -20,11 +20,23 @@ class InvalidBrokerIdError(KafkaToolError):
     pass
 
 
+class InvalidBrokerError(KafkaToolError):
```

Comment:
InvalidBrokerError seems a bit of a broad error. We can either just use assertion or rename this exception as BrokerNotInReplicationGroup error. However, this condition would happen only if there was a code bug in the algorithm itself. In these cases I usually prefer to use "assert".